--- a/launchk/src/main.rs
+++ b/launchk/src/main.rs
@@ -12,6 +12,5 @@
 use cursive::view::Resizable;
 use cursive::views::{NamedView, Panel};
-use git_version::git_version;
 use std::process::exit;

 use crate::launchd::plist::{init_plist_map, PLIST_MAP_INIT};
@@ -38,7 +37,7 @@
     let root_layout = RootLayout::new(&mut siv, runtime.handle());
     let root_layout = NamedView::new("root_layout", root_layout);

-    let panel = Panel::new(root_layout)
-        .title(format!("launchk ({})", git_version!()))
+    let panel = Panel::new(root_layout)
+        .title("launchk")
         .full_width()
         .full_height();
